api_tests/files/serializers/test_file_serializer.py

```diff
@@ -1,48 +1,66 @@
 # -*- coding: utf-8 -*-
+import pytest
 from datetime import datetime
-from nose.tools import *  # flake8: noqa
 from pytz import utc
 
-from osf_tests import factories
+from osf_tests.factories import UserFactory, NodeFactory
 from tests.base import DbTestCase
 from tests.utils import make_drf_request_with_version
-
 from api.files.serializers import FileSerializer
-
 from api_tests import utils
 
+@pytest.fixture()
+def user():
+    return UserFactory()
+
+@pytest.mark.django_db
+class TestFileSerializer:
+
+    @pytest.fixture()
+    def node(self, user):
+        return NodeFactory(creator=user)
+
+    @pytest.fixture()
+    def file(self, node, user):
+        return utils.create_test_file(node, user)
+
+    @pytest.fixture()
+    def date_created(self, file):
+        return file.versions.first().date_created
 
-class TestFileSerializer(DbTestCase):
+    @pytest.fixture()
+    def date_modified(self, file):
+        return file.versions.last().date_created
 
-    def setUp(self):
-        super(TestFileSerializer, self).setUp()
-        self.user = factories.UserFactory()
-        self.node = factories.NodeFactory(creator=self.user)
-        self.file = utils.create_test_file(self.node, self.user)
+    @pytest.fixture()
+    def date_created_tz_aware(self, date_created):
+        return date_created.replace(tzinfo=utc)
 
-        self.date_created = self.file.versions.first().date_created
-        self.date_modified = self.file.versions.last().date_created
-        self.date_created_tz_aware = self.date_created.replace(tzinfo=utc)
-        self.date_modified_tz_aware = self.date_modified.replace(tzinfo=utc)
+    @pytest.fixture()
```

Comment:
Drive-by comment: no need to make these date strings fixtures if they're only going to be used in one test.